packages/devextreme/js/localization/messages/zh-tw.json

```
@@ -322,6 +322,14 @@
         "dxCalendar-selectedDateRangeCount": "There are {0} selected date ranges",
         "dxCalendar-readOnlyLabel": "Read-only calendar",
 
+        "dxAvatar-defaultUserName": "Unknown User",
+        "dxAvatar-imageAlt": "Avatar",
```

Shouldn't it be by analogy with `"dxAvatar-defaultUserName"`?
```suggestion
        "dxAvatar-defaultImageAlt": "Avatar",
```

---

[aa1c45fcfe057ea307c9912fa5e14fa0c4a5a231]